backend/main.go

```diff
@@ -72,6 +92,14 @@ func init() {
 	}
 }
 
+func newKubeconfig(kubeconfig string) (*rest.Config, error) {
```

Comment:
Use the `clientcmd` package loaders (e.g. [this one](https://pkg.go.dev/k8s.io/client-go/tools/clientcmd#NewNonInteractiveDeferredLoadingClientConfig)) to model the chain of loaders you want to use in the order you want them. Easy to set up `$KUBECONFIG > --kubeconfig > in-cluster config`

Reply:
Update 1st commit.